src/neo/Network/P2P/Payloads/Witness.cs

```diff
@@ -24,6 +26,8 @@ public virtual UInt160 ScriptHash
             }
         }
 
+        public bool StateDependent => VerificationScript.Length == 0;
```

Comment:
We only allow to read the state if the script it's stored?

Reply:
Yes. I think this is a good idea.